shell/platform/windows/system_utils_winuwp.cc

```diff
@@ -8,6 +8,9 @@
 
 #include <sstream>
 
+#include <third_party/cppwinrt/generated/winrt/Windows.Foundation.Collections.h>
```

Comment:
Nit: This is not a system header. Please use use quotes instead of angle brackets to include these.

Reply:
done